src/UnwindCursor.hpp

```diff
@@ -2697,7 +2697,13 @@ bool UnwindCursor<A, R>::setInfoForSigReturn(Registers_s390x &) {
   // own restorer function, though, or user-mode QEMU might write a trampoline
   // onto the stack.
   const pint_t pc = static_cast<pint_t>(this->getReg(UNW_REG_IP));
-  const uint16_t inst = _addressSpace.get16(pc);
+  uint16_t inst = 0;
+
+  struct iovec local_iov = {&inst, sizeof(inst)};
+  struct iovec remote_iov = {reinterpret_cast<void *>(pc), sizeof(inst)};
+  long bytesRead =
```

Comment:
There should be ifdef to check for Linux.

Reply:
Since the code is under
 #if defined(_LIBUNWIND_CHECK_LINUX_SIGRETURN)
and _LIBUNWIND_CHECK_LINUX_SIGRETURN is defined at line 38, which is under _LIBUNWIND_TARGET_LINUX, so there is no need to check for Linux.